Branch2018/test1/Product.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace test1
+{
+    public class Product
+    {
+        public Product()
+        {
+        }
+
+        public int ProductId { get; set; }
+
+        public string NameProduct { get; set; }
```

Comment:
The right way will be ProductName, ProductDescription etc at all class and also these kind of class could be better inside their own namespace or sub namespace.

Reply:
ready